.github/workflows/build_docs.yaml

```diff
@@ -0,0 +1,45 @@
+name: Build docs and upload them to gh-pages
+
+on:
+  push:
+    branches:
+      - gh-pages-docs
```

Comment:
This is just while I'm testing, it should be changed to `master`, or even to listen to tagged commits.